app/web/server.rb

```diff
@@ -41,28 +41,28 @@ def json_params
     post '/todo/:todo_id' do
       command = Commands::Todo::Add::Command.new(json_params)
       command.valid?
-      Commands::Todo::Add::CommandHandler.handle(command)
+      Commands::Todo::Add::CommandHandler.new.handle(command)
       status 201
     end
 
     put '/todo/:todo_id' do
       command = Commands::Todo::Amend::Command.new(json_params)
       command.valid?
```

Comment:
I'm going to merge this as is and we can take on handler refactoring separate. @andrewgr we're aiming for this to be educational so I'd say anything inline that is obvious is probably the sweet spot.